main.gd

```diff
@@ -25,6 +25,33 @@ func _ready():
 				child.set_sprite("res://art/dome_sprites/Dome_science_96.png")
 				
 	$Domes/Dome.set_sprite("res://art/dome_sprites/Dome_hq_96.png")
+	
+	# try to put the lower number dome on the left to help prevent repeats, and sort ascending
+	var dome_connections = [
```

Comment:
looks verbose, but we shouldn't need to edit often. If it's a pain we can convert to just numbers that get translated to node refs, but this is less error-prone imo. find+replace is your friend :)